doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -89,6 +91,18 @@ Bluetooth libraries and services
 
 * |no_changes_yet_note|
 
+Libraries for networking
+------------------------
+
+* :ref:`lib_location` library:
+
+  * Added A-GPS filtered ephemerides support.
+
+* :ref:`lib_nrf_cloud` library:
+
+  * Added A-GPS filtered ephemerides support, with ability to set matching threshold mask angle.
+  * When filtered ephemerides is enabled, limit A-GPS assistance requests to cloud to no more than once every two hours.
```

Comment:
Maybe instead of imperative: ..., A-GPS assistance requests to cloud are limited to...